.github/workflows/stale.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v2.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # See https://github.com/actions/stale/blob/master/action.yml for description of the options
+
+          stale-issue-label: unloved
+
+          stale-issue-message: |
+            This issue is marked as unloved because it has not had any activity for 180 days.
+
+            It doesn't mean it's not important, so please remove the unloved label if you like it, or add a comment saying what it means to you :)
```

Comment:
Could add something about "if this was a big, please test to see if this is still an issue?"